pkg/types/openstack/defaults/platform.go

```diff
@@ -1,9 +1,37 @@
 package defaults
 
 import (
+	"fmt"
+
+	"github.com/apparentlymart/go-cidr/cidr"
+
+	"github.com/openshift/installer/pkg/types"
 	"github.com/openshift/installer/pkg/types/openstack"
 )
 
+// Defaults for the openstack platform.
+const (
+	APIVIP = ""
+	DNSVIP = ""
+)
+
 // SetPlatformDefaults sets the defaults for the platform.
-func SetPlatformDefaults(p *openstack.Platform) {
+func SetPlatformDefaults(p *openstack.Platform, c *types.InstallConfig) {
```

Comment:
installconfig has the platform.. i think it's better to pass the networking.

Reply:
We just did the same thing baremetal and vsphere did:

https://github.com/openshift/installer/blob/fc8c68319948411e384005a2ca55266c5fd0e200/pkg/types/baremetal/defaults/platform.go#L23

https://github.com/openshift/installer/blob/fc8c68319948411e384005a2ca55266c5fd0e200/pkg/types/defaults/installconfig.go#L72-L74

But I'll pass `Networking` in directly.